### PR TITLE
[WIP] new old APC sprites

### PR DESCRIPTION
**Pull request by pubby**
A repainting of the old APC sprites to look like the Eris sprites except not shitty.

Still a work in progress.

Closes #29716 
Closes #29901

image

image
:cl: Pubby
imageadd: new APC sprites (again)
/:cl:

[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:)

### CODE REVIEW

**Comment by GunHog** [comment]
They just look smaller now. APCs should not be smaller given all the information provided from the animation alone! (Charging state, emag state, breaker on/off, equip/power/atmo on/off/auto, locked/unlocked)

**Comment by pubby** [comment]
These are 1 pixel thinner than the old sprites and have the same indicator lights.

I do not understand what you are talking about.

**Comment by GunHog** [comment]
Old APCs were shaped like rectangles, yours look like squares. It looks scrunched in comparison.

**Comment by pubby** [comment]
image

This one is now exactly the same dimensions as the old one.

**Comment by GunHog** [comment]
Works for me :+1:

**Comment by WJohn** [comment]
These don't even get the benefits of the new ones, and actually are less readable than the old ones. 👎 from me.

**Comment by pubby** [comment]
Higher contrast diode indicators: yay or nay?

image

**Comment by Hatfish** [comment]
Better than what we have now 👍

**Comment by WJohn** [comment]
I mean to say that they don't have the vertically laid out lights, which is the benefit I'm talking about. Besides that, I rather like 3/4th and am biased towards it. I think we could move more towards it. We'll see if there are still complaints after the other contrast update PR is merged.

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -120,8 +120,8 @@
 	if(auto_name)
 		name = "\improper [get_area(src)] APC"
 
-	pixel_x = (src.tdir & 3)? 0 : (src.tdir == 4 ? 24 : -25)
-	pixel_y = (src.tdir & 3)? (src.tdir ==1 ? 23 : -24) : 0
+	pixel_x = (src.tdir & 3)? 0 : (src.tdir == 4 ? 24 : -24)
+	pixel_y = (src.tdir & 3)? (src.tdir ==1 ? 24 : -24) : 0
```
Comment (Cyberboss) [inline code/modules/power/apc.dm]: remove `src`

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Dorsisdwarf** [comment]
I like how these look far more than either the current ones or the other option

**Comment by Qbopper** [comment]
looks really nice, much better than the current new one

**Comment by pubby** [comment]
I've added an upper edge to make it a subtle 3/4 perspective and redid the indicator lights to make them readable. I did not explain this before, but the lock/unlock indicator is animated into the card reader.

image

image

I'll consider this my final version. If @WJohn doesn't approve, I'll close this.

**Comment by nicbn** [comment]
This no longer closes #29716.

**Comment by optimumtact** [comment]
#29931 is the blessed version

**Pull request closed without merging**